index.html

```diff
@@ -60,26 +70,34 @@ <h3 class="feature-item-title">Настоящий городок</h3>
                     </ul>
                 </li>
                 <li class="feature-item blue">
-                    <h3 class="feature-item-title">Там есть <br>мост дьявола</h3>
-                    <span class="feature-item-text">-№2-</span>
-                    <p class="feature-item-text">Да, по нему можно пройти! Если, конечно, вы осмелитесь</p>
-                    <img src="img/devil_bridge.jpg" width="800" height="256" alt="мост дьявола">
-                </li>
-                <li class="feature-item grey">
-                    <h3 class="feature-item-title">Небольшая площадь</h3>
-                    <span class="feature-item-text">-№3-</span>
-                    <p class="feature-item-text">Все достопримечательности находятся очень близко</p>
-                </li>
-                <li class="feature-item grey">
-                    <h3 class="feature-item-title">Красивая дорога</h3>
-                    <span class="feature-item-text">-№4-</span>
-                    <p class="feature-item-text">Еехать в Седону из Лас-Вегаса совсем не скучно!</p>
-                </li>
-                <li class="feature-item grey">
-                    <h3 class="feature-item-title">Мало туристов</h3>
-                    <span class="feature-item-text">-№5-</span>
-                    <p class="feature-item-text">Большинство едет В Гранд Каньон и толпится там</p>
+                    <div class="feature-bar-sc">
+                        <div class="inner-text-sc">
+                            <h3 class="feature-item-title">Там есть <br>мост дьявола</h3>
+                            <span class="feature-item-text">-№2-</span>
+                            <p class="feature-item-text">Да, по нему можно пройти! Если, конечно, вы осмелитесь</p>
+                        </div>
+                        <div class="bridge-pic">
```

Comment:
класс должен быть тот же, что и для предыдущей картинки, так как стилизация у них одинаковая:
```suggestion
                        <div class="feature-picture">
```